### PR TITLE
Feature/gh actions

### PR DESCRIPTION
**Pull request by eheinrich**
Switch from using TravisCI to using GitHub Actions.

### CODE REVIEW

**Review by jnation3406 — APPROVED** [review]

**Comment by jnation3406** [comment]
I'm surprised this works without adding the gfortran and stuff...

**Comment by eheinrich** [comment]
Yeah, the [environment the tests are running in](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu1804-README.md) has gfortran installed already. That page doesn't say anything explicitly about the lapack or blas libraries being pre-installed, but yeah, seems to work...

File: .travis.yml [hunk 1]
```diff
@@ -1,34 +0,0 @@
...
-before_deploy:
-- echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
-- export IMAGE=observatorycontrolsystem/adaptive_scheduler:${TRAVIS_TAG}
-- docker build -t $IMAGE .
-deploy:
-- provider: script
-  script: docker push $IMAGE
-  skip_cleanup: true
-  on:
-    tags: true
```
Comment (jnation3406) [inline .travis.yml:34]: newline

Comment (eheinrich) [reply .travis.yml:34]: this file is being deleted, so the absence of a newline is going away

Comment (jnation3406) [reply .travis.yml:34]: lol i have baby brain

**Pull request merged**